trp_use_case/model/use_case.py

```diff
@@ -205,3 +215,5 @@ def _get_hours_total(self):
         compute=_get_hours_total, string="Total nr. of optional hours")
     hours_total_nonoptional = fields.Float(
         compute=_get_hours_total, string="Total nr. of non optional hours")
+    tot_use_cases = fields.Integer(
+        compute=_get_hours_total, string="tot_use_cases")
```

Comment:
`string` should always be a human readable string, I assume this is meant for the use case count?